includes/Admin/Settings_Screens/Commerce.php

```diff
@@ -68,9 +68,25 @@ public function render() {
 	 * @internal
 
 	 * @since 2.1.0-dev.1
+	 *
+	 * @param array $field field data
 	 */
-	public function render_google_product_category_field() {
-
+	public function render_google_product_category_field( $field ) {
+
+		$category_field = new Admin\Google_Product_Category_Field();
```

Comment:
It felt weird to do that. Should this class be abstract and contain static methods instead?